core/edit/delimiter_pair.py

```diff
@@ -0,0 +1,32 @@
+from talon import Module, actions
+
+mod = Module()
+
+mod.list("delimiter_pair", "List of matching pair delimiters")
+
+
+@mod.capture(rule="{user.delimiter_pair}")
+def delimiter_pair(m) -> list[str]:
+    pair = m.delimiter_pair.split()
+    assert len(pair) == 2
+    open = pair[0] if pair[0] != "space" else " "
+    close = pair[1] if pair[1] != "space" else " "
+    return [open, close]
```

Comment:
```suggestion
    # "space" requires a special written form because Talon lists are whitespace insensitive
    open = pair[0] if pair[0] != "space" else " "
    close = pair[1] if pair[1] != "space" else " "
    return [open, close]
```

(Feel free to rewrite, I don't fully understand the issue yet)